src/components/navigation/SecondaryNavBar.tsx

```diff
@@ -12,11 +12,11 @@ const SecondaryNavBar: FC<{ navItems: NavItem[] }> = ({ navItems }) => {
     'no-underline hover:decoration-clementine hover:underline hover:underline-offset-8 hover:decoration-[0.2rem]',
   )
   return (
-    <nav className="center sticky top-0 z-40 bg-darkTeal py-2.5">
+    <nav className="center sticky top-0 z-40 hidden bg-darkTeal py-2.5 sm:block">
```

Comment:
```suggestion
    <nav className="center sticky top-0 z-40 hidden bg-darkTeal py-2.5 md:block">
```

If we keep it hidden until medium size screen then it will appear when the primary nav bar appear which is nice continuity.

Reply:
mm good catch ty